### PR TITLE
site: Fix command name, missing letter in table header and deduplicate

### PR DESCRIPTION
**Pull request by jhutar**

### CODE REVIEW

File: site/content/en/docs/handbook/accessing.md [hunk 1]
```diff
@@ -53,13 +53,10 @@ Services of type `NodePort` can be exposed via the `minikube service <service-na
...
     ```shell
-    kubectl get svc
-    ```
-    <pre>
-    $ kc get svc
-    AME              TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+    $ kubectl get svc
+    NAME              TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
     hello-minikube1   NodePort    10.100.238.34   <none>        8080:31389/TCP   3s
-    </pre>
```
Comment (medyagh) [inline site/content/en/docs/handbook/accessing.md:62]: Thanks for the fix, kc was actually my alias for kubectl lol

however why you are removing /pre?

**Review by medyagh — CHANGES_REQUESTED** [review]

File: site/content/en/docs/handbook/accessing.md (hunk 1 above)
Comment (jhutar) [reply site/content/en/docs/handbook/accessing.md:62]: Hello! I'm just trying to align to e.g. this below:
```
    ```shell
    $ ps -ef | grep docker@127.0.0.1
    ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -N docker@127.0.0.1 -p 55972 -i /Users/FOO/.minikube/machines/minikube/id_rsa -L TUNNEL_PORT:CLUSTER_IP:TARGET_PORT
    ```
```
Also it looked like it is duplicating the command.

I did not gave it too much of recommended style read / thought :)

Comment (spowelljr) [reply site/content/en/docs/handbook/accessing.md:62]: It's already in a code block so I don't think `<pre>` is needed

**Review by spowelljr — APPROVED** [review]
Thanks for the fixes

**Pull request merged**